flask_admin/model/base.py

```diff
@@ -1477,6 +1477,10 @@ def create_view(self):
             return redirect(return_url)
 
         form = self.create_form()
+        if self._form_create_rules:
```

Comment:
I'm thinking if it should be really dynamic. Can do as a part of the form scaffolding. If someone will want dynamic form - they can always do it later.

Also, please move this logic into separate method, so it is callable and overridable.